feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -102,18 +110,65 @@ fun BookmarksScreen(
                     item(span = { GridItemSpan(maxLineSpan) }) {
                         Spacer(Modifier.windowInsetsBottomHeight(WindowInsets.safeDrawing))
                     }
-                } else item(span = { GridItemSpan(maxLineSpan) }) {
-                    Box(
-                        modifier = Modifier
-                            .fillMaxHeight()
-                            .wrapContentSize()
-                            .testTag("bookmarks:empty"),
-                        contentAlignment = Alignment.Center
-                    ) {
-                        Text(text = stringResource(id = R.string.bookmarks_empty))
-                    }
                 }
             }
         }
     }
+
+    /**
+     ** The [Column] is handling the Empty state when the feed is empty.
+     **/
+    if (feedState is NewsFeedUiState.Success && feedState.feed.isEmpty()) {
+        OnEmptyStateView()
+    }
 }
+
+@Composable
+private fun OnEmptyStateView() {
```

Comment:
A Composable function should not have an `on` name, consider renaming to "EmptyState()"